testsuite/tests/init/with-chdir/test.py

```diff
@@ -0,0 +1,20 @@
+"""
+Test "executable" only appears in --bin initializations
+"""
```

Comment:
Quite common to forget to update this description ;-)

Reply:
Thanks for catching that - fixed in cba2c11, which also tests -C behavior on a non-existent directory. Thanks!